<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundfontViewerLeft</class>
 <widget class="QWidget" name="SoundfontViewerLeft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="iconAuthor">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelAuthor">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string notr="true">labelAuthor</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="iconDate">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelDate">
     <property name="text">
      <string notr="true">labelDate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="iconLicense">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelLicense">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string notr="true">labelLicense</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="iconWebsite">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelWebsite">
     <property name="text">
      <string notr="true">labelWebsite</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelCategory">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="AttributeFlow" name="attributeCategory" native="true"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="labelSampleSource">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample source</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="AttributeFlow" name="attributeSampleSource" native="true"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="labelTimbre">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Timbre</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="AttributeFlow" name="attributeTimbre" native="true"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="labelArticulation">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Articulation</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="AttributeFlow" name="attributeArticulation" native="true"/>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="labelGenre">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Genre</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="AttributeFlow" name="attributeGenre" native="true"/>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QLabel" name="labelMidiStandard">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MIDI standard</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="AttributeFlow" name="attributeMidiStandard" native="true"/>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QLabel" name="labelTags">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="2">
    <widget class="AttributeFlow" name="attributeTag" native="true"/>
   </item>
   <item row="19" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AttributeFlow</class>
   <extends>QWidget</extends>
   <header>attributeflow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
